pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_user-roles.adoc

```diff
@@ -1,5 +1,5 @@
 [id="user-roles_{context}"]
-= User roles
+= Pantheon user roles
```

Comment:
@dvagnero 
We use the attribute {product-name} wherever possible. We're trying to keep this as upstream-friendly as possible. Using the attribute for the product name supports this effort.

Reply:
Yes, sure. I'll search and replace all the instances